keps/sig-storage/5030-attach-limit-autoscaler/kep.yaml

```
@@ -0,0 +1,50 @@
+title: Integrate CSI Volume attach limits with cluster autoscaler
+kep-number: 5030
+authors:
+  - "@gnufied"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+  - sig-scheduling
+  - sig-autoscaling
+status: provisional
```

To satisfy the requirement for Enhancement Freeze, the status must be ready to move to:

```suggestion
status: implementable
```